backend/src/main/java/gov/cdc/usds/simplereport/db/repository/PatientLinkRepository.java

```diff
@@ -13,7 +13,7 @@ public interface PatientLinkRepository extends EternalAuditedEntityRepository<Pa
 
   @Query(
       value =
-          "select DISTINCT on (ordered.test_order_id) * from (select * from {h-schema}patient_link where test_order_id in :testOrderIds order by created_at desc ) ordered where test_order_id in :testOrderIds",
+          "select DISTINCT on (ordered.test_order_id) * from (select * from {h-schema}patient_link where test_order_id in :testOrderIds order by created_at desc ) ordered",
```

Comment:
removing extra where clause per @mehansen feedback, discussed multiple implementations and sub-querys tended to perform the best

Reply:
when we removed the outer where, didn't it make it just as slow as removing the subquery though? I thought no outer where and no subquery ended up being about the same for some weird reason

Reply:
yes it did, but not after I added the index!

Reply:
This has been moved to here https://github.com/CDCgov/prime-simplereport/pull/4319 per @emmastephenson feedback